app/styles/modules/lustrum.scss

```diff
@@ -0,0 +1,279 @@
+$lustrum-navyblue: #1e2759;
+$lustrum-steelblue: #548897;
+$lustrum-yellow: #ffc10b;
+$lustrum-red: #f26b6b;
+$lustrum-white: #f2f2f2;
+
+.app.lustrum {
+  margin-bottom: -50px;
+  background-color: $lustrum-white;
+  padding-bottom: 100px;
+  font-family: 'Raleway', $font-family-sans-serif; /* stylelint-disable-line font-family-no-missing-generic-family-keyword */
+}
+
+.lustrumcard {
+  margin-bottom: 50px;
+
+  .lustrumcard-header {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .lustrumcard-logo {
+    margin: 40px;
+    margin-left: 0;
+    max-height: 250px;
+  }
+
+  .lustrumcard-titles {
+    margin: 50px 0;
+  }
+
+  .lustrumcard-title {
+    margin: 0;
+    line-height: 100px;
+    color: $lustrum-navyblue;
+    font-size: 110px;
+    font-weight: 500;
+  }
+
+  .lustrumcard-subtitle {
+    margin: 0;
+    color: $lustrum-steelblue;
+    font-size: 50px;
+    font-weight: 200;
+  }
+
+  .lustrumcard-line {
+    margin-top: -40px;
+    margin-bottom: 20px;
+    width: 100%;
+  }
+
+  .lustrumcard-text {
+    text-align: justify;
+    font-size: 16px;
+
+    &.text-enlarged {
+      font-size: 18px;
+      font-weight: 500;
+    }
+  }
+}
+
+.lustrumactivities {
+  .lustrumactivities-title {
+    font-size: 40px;
+  }
+}
+
+.lustrumbar {
+  overflow: hidden;
+
+  &.lustrumbar-expandable {
+    .lustrumbar-header:hover,
+    .lustrumbar-button:hover {
+      cursor: pointer;
+    }
+  }
+
+  &.lustrumbar-steelblue .lustrumbar-header {
+    background-color: $lustrum-steelblue;
+  }
+
+  &.lustrumbar-navyblue .lustrumbar-header {
+    background-color: $lustrum-navyblue;
+  }
+
+  &.lustrumbar-yellow .lustrumbar-header {
+    background-color: $lustrum-yellow;
+    color: #000;
+  }
+
+  &.lustrumbar-red .lustrumbar-header {
+    background-color: $lustrum-red;
+  }
+
+  &.lustrumbar-red,
+  &.lustrumbar-yellow {
+    .lustrumbar-button {
+      background-color: $lustrum-steelblue;
+    }
+  }
+
+  &.lustrumbar-navyblue,
+  &.lustrumbar-steelblue {
+    .lustrumbar-button {
+      background-color: $lustrum-red;
+    }
+  }
+
+  .lustrumbar-header {
+    transform: rotate(-2deg);
+    margin: 20px -10px 0;
+    padding: 20px 40px;
+    color: $lustrum-white;
+  }
+
+  .lustrumbar-titles {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    > * {
+      margin: 0;
+      font-size: 30px;
+    }
+  }
+
+  .lustrumbar-block {
+    display: flex;
+    align-items: self-start;
+    justify-content: space-between;
+    transform: rotate(-2deg);
+    padding: 5px 25px;
+  }
+
+  .lustrumbar-button {
+    transform: rotate(2deg);
+    margin-top: -20px;
+    margin-left: 20px;
+    border: solid 5px $lustrum-white;
+    border-radius: 30px;
+    padding: 10px 15px;
+    text-align: center;
+    color: $lustrum-white;
+    font-weight: bold;
+  }
```

Comment:
If you add `white-space: nowrap;` here, the text 'Schrijf je in!' won't be split over two lines, looks a bit nicer in my opinion. Don't know if you were finished with that design part yet or not :sweat_smile:

Reply:
Ow, dat is wel een goeie. Ik vond het niet heel erg.